app/models/manageiq/providers/vmware/infra_manager.rb

```diff
@@ -112,6 +112,20 @@ def self.params_for_create
                           :isRequired => true,
                           :validate   => [{:type => "required"}]
                         },
+                        {
+                          :component    => 'text-field',
+                          :id           => 'endpoints.default.port',
+                          :name         => 'endpoints.default.port',
+                          :label        => _('Port (default 443)'),
```

Comment:
Is this string consistent with other providers? (I don't recall the default being in the label elsewhere)

Reply:
Let me check, actually the default in the label doesn't make sense because it'll be in the text box

Reply:
The rest use "API Port", will change to that.

Reply:
Done